tdrs-backend/docker-compose.yml

```diff
@@ -5,13 +5,22 @@ services:
   zaproxy:
     image: owasp/zap2docker-weekly
     command: sleep 3600
+
   postgres:
     image: postgres:11.6
     environment:
       - POSTGRES_USER=tdpuser
       - POSTGRES_PASSWORD=something_secure
       - POSTGRES_DB=tdrs_test
       - POSTGRES_PORT=5432
+
+  clamav-rest:
+    image: rafttech/clamav-rest:0.103.2
```

Comment:
Uses a forked image that has been updated to the latest ClamAV. This turned out to be necessary due to a required update on the ClamAV side that made all previous versions incompatible with the automated signature updates performed by `freshclam`.

https://hub.docker.com/r/rafttech/clamav-rest